rautomation.gemspec

```diff
@@ -17,6 +17,7 @@ RAutomation provides:
   s.homepage = %q{http://github.com/jarmo/RAutomation}
   s.summary = %q{Automate windows and their controls through user-friendly API with Ruby}
   s.license = "MIT"
+  s.platform = Gem::Platform.local if s.platform == 'ruby'
```

Comment:
What does this line do?

Reply:
With this if statement it is possible to force a specific platform using the --platform parameter. Eg 'gem build rautomation.gemspec --platform x86' 

This is however not relevant any longer since I edited the function that retrieves the current Ruby platform.